python/ray/serve/backend_state.py

```diff
@@ -91,9 +94,18 @@ def __set_state__(self, d: Dict[Any, Any]) -> None:
         self._ready_obj_ref = None
         self._drain_obj_ref = None
 
+    @property
+    def replica_tag(self) -> str:
+        return str(self._replica_tag)
+
+    @property
+    def backend_tag(self) -> str:
+        return self._backend_tag
+
     @property
     def actor_handle(self) -> ActorHandle:
-        return ray.get_actor(self._actor_name)
+        return ray.get_actor(
+            self._actor_name, namespace=self._controller_namespace)
```

Comment:
Is this necessary? I don't think it is because it should default to current namespace anyways.

Reply:
im honestly not totally sure, but one of the commit was about changing actor naming since currently each replica is also named as "SERVE_CONTROLLER_ACTOR:Deployment#zxs12" where the first part should be removed. So i ensured all actor calls are using the same controller namespace so we no longer need to append "SERVE_CONTROLLER_ACTOR:Deployment" in `format_actor_name`